src/main/java/Duke.java

```diff
@@ -94,14 +96,24 @@ public static void main(String[] args) {
                         break;
                     case MARK:
```

Comment:
Do make sure, your `switch` and `case` keywords lie under the same indentation line 👌

Reply:
Good catch. Will update in future version.